### ISSUE TITLE
expo start --config no longer respecting config location

### ISSUE THREAD

**Issue by bayoremit** [expo/expo-cli#1785]
## Description

I have my app.json in a custom location and start expo w/ `expo start --clear --config ./app_configs/app.preprod.json`. This used to work in cli v3.16.xx and below. Now It fails as its looking at the default location and errors out.

## Expected Behavior

Expo will start as usual and pull its app manifest file from the custom location

## Observed Behavior

Expo tries to start and errors out
```
Error: Problems validating fields in app.json. See https://docs.expo.io/versions/v36.0.0/workflow/configuration/
 • is missing required property 'name'.
 • is missing required property 'slug'.
```

## Environment

```
  Expo CLI 3.17.8 environment info:
    System:
      OS: macOS 10.15.3
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
      Yarn: 1.21.1 - /usr/local/bin/yarn
      npm: 6.14.2 - ~/.nvm/versions/node/v10.16.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: 11.4/11E146 - /usr/bin/xcodebuild
    npmPackages:
      expo: 36.0.2 => 36.0.2 
      react: 16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4 
    npmGlobalPackages:
      expo-cli: 3.17.8
```

Also specify:

- Expo web
- Iphone simulator 12.1

## Reproducible Demo

- Move app.json file somewhere other than the project root e.g. `./manifest/app.json`
- Run `expo start --config ./manifest/app.json` on expo-cli v 3.17.8
- Do NOT PROFIT.... Sadness all around 😢

Thanks Team Expo!!!

**Comment by EvanBacon** [comment]
investigating...

**Comment by EvanBacon** [comment]
Package versions got out of sync while we were fixing bugs after the SDK37 release yesterday. Sorry for the inconvenience, we'll publish after merging #1787.

**Comment by bayoremit** [comment]
Wooo, Super thanks. We await the next release

**Comment by EvanBacon** [comment]
Fixed in `expo-cli@3.17.9`. Please run `npm i -g expo-cli` to get the latest. Again, sorry for any inconvenience this may have caused.
